tests/tests.py

```diff
@@ -1038,66 +1044,42 @@ def test_plantcv_transform_warp_smaller():
     bimg_small = cv2.resize(bimg, (200,300)) #not sure why INTER_NEAREST doesn't preserve values
     bimg_small[bimg_small>0]=255
     mrow, mcol = bimg_small.shape
-    vrow, vcol, vdepth = img.shape
-    pcv.params.debug = None
-    mask_warped = pcv.transform.warp(bimg_small, img[:,:,2],
-                                    pts = [(0,0),(mcol-1,0),(mcol-1,mrow-1),(0,mrow-1)],
-                                    refpts = [(0,0),(vcol-1,0),(vcol-1,vrow-1),(0,vrow-1)])
-    pcv.params.debug = 'plot'
-    mask_warped_plot = pcv.transform.warp(bimg_small, img[:,:,2],
-                                pts = [(0,0),(mcol-1,0),(mcol-1,mrow-1),(0,mrow-1)],
-                                refpts = [(0,0),(vcol-1,0),(vcol-1,vrow-1),(0,vrow-1)])
-
-    assert np.count_nonzero(mask_warped)==93142
```

Comment:
good to get rid of the tests with precomputed answers per yesterday's conversation. I think this function would lend itself well to parametrize the tests  and we test all combinations of inputs wrt rgb and binary since they could have slightly different behaviors in the future due to # dimensions.